docs/source/quick-start/opf.rst

```diff
@@ -34,7 +34,7 @@ using the :class:`.ModelFactory`.
 
 .. literalinclude:: ../../examples/opf/create-model-example.py
 
-The resulting ``model`` will be an instance of :class:`.CLAModel`.
+The resulting ``model`` will be an instance of :class:`.SPTMModel`.
```

Comment:
@subutai - "Prediction" and "streaming" aren't good because all OPF models are streaming and prediction. One really important thing to capture is that this is HTM-related since other models may be for reference (like TwoGramModel). While "HTMModel" might be ok, we may split out certain variants into their own classes rather than shoe-horning all HTM-variants into a single OPF model.

That is how we got to "SPTMModel" but definitely open to other ideas.